src/main/java/io/vertx/core/http/HttpServerOptions.java

```diff
@@ -344,6 +352,24 @@ public HttpServerOptions setCompressionSupported(boolean compressionSupported) {
   }
 
   /**
+   * @return the server gzip compression level
+   */
+  public int getCompressionLevel() {
+    return this.compressionLevel;
+  }
+  
+  /**
+   * Set whether the server supports compression
```

Comment:
this is wrong, this set the compression level, compression is set by HttpServerOptions#setCompressionSupported(true)